tests/common.py

```diff
@@ -24,23 +24,22 @@
         json.dump(service_accounts, fh)
 
 
-def create_test_statement(name: str):
+def create_test_statement(name: str, actions: typing.List[str] = None, JSON=False):
```

Comment:
nit: the `JSON` parameter is potentially confusing. Consider dropping it and just serializing on the client side when necessary.